### ISSUE TITLE
Self-closing tags (in __layout.svelte) are ruined

### ISSUE THREAD

**Issue by bato3** [svelte-add/svelte-add#127]
Hi!

When I start from [sveltekit-electron](https://github.com/FractalHQ/sveltekit-electron) In `__layout.svelte` construcion 
```html

<div class="dragbar" />

{#if ready}
	<slot />
{/if}
<style>.foo{}</style>

```
 is changed to 
```html
<div class="dragbar">

{#if ready}
	<slot>
{/if}
<style>.foo{}</style>
</slot></div>
<slot></slot>
```

**Comment by babichjacob** [comment]
Note to self: this probably requires switching away from `posthtml-parse` and `posthtml-render` to a different XML / HTML parsing + stringifying library.

**Comment by manuel3108** [comment]
I don't know whats the problem here, but maybe we could use the svelte-parser for `.svelte` files, see https://svelte.dev/docs#svelte_parse

**Comment by babichjacob** [comment]
> I don't know whats the problem here, but maybe we could use the svelte-parser for `.svelte` files, see https://svelte.dev/docs#svelte_parse

I don't know how (or if) you can stringify the AST this creates back to HTMLX.

**Comment by manuel3108** [comment]
> I don't know how (or if) you can stringify the AST this creates back to HTMLX.

According to https://github.com/sveltejs/svelte/issues/2914 this wont work anyway. So we will need to find another way.

**Comment by babichjacob** [comment]
Thanks for the report and patience! This has been fixed by an internal refactoring in f29c47d991cc8f115514d33d9d75b56ac22eed6c.
